src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/AssemblyReference.xaml

```diff
@@ -17,13 +17,12 @@
                       DisplayName="Aliases"
                       Separator="," />
 
-  <BoolProperty Name="CopyLocal"
+  <BoolProperty Name="Private"
                 Description="Indicates whether the reference will be copied to the output directory."
                 DisplayName="Copy Local">
     <BoolProperty.DataSource>
       <DataSource HasConfigurationCondition="False"
                   ItemType="Reference"
-                  PersistedName="Private"
```

Comment:
The `PersistedName` is supposed to control what actually gets written to the project file. So while this change effectively works around the problem, I suspect there is a deeper bug here.

Reply:
Agreed. We have many other usages of `PersistedName`, however all on properties that are either non-visible or read only. As far as I can tell, they are all working correctly. I do not know whether this property ever worked correctly.

@tmeschter do you have a sense of which component might be at fault here?